### ISSUE TITLE
Ukrainian translation

### ISSUE THREAD

**Issue by chefranov** [epinter/tqrespec#23]
Hi! Could you provide actual English strings? I would like to translate into Ukrainian. Before I translated into Russian.

**Comment by epinter** [comment]
Hi

They are here: https://github.com/epinter/tqrespec/tree/master/src/main/resources/i18n

The UI.properties contains the english strings.

Thanks!

**Comment by chefranov** [comment]
Hey! Sorry for delay, I forgot 😅
```
#
# Copyright (C) 2019 Emerson Pinter - All Rights Reserved
# Ukrainian translate: Evgeniy Chefranov (titanquest.org.ua)
#

main.translators=Перекладач:\nEvgeniy Chefranov\n
main.strength=Сила
main.intelligence=Інтелект
main.dexterity=Спритність
main.health=Здоров`я
main.energy=Енергія
main.savechar=Зберегти
main.copychar=Копія як
main.copycharname=Новий персонаж
main.available=Доступні очки
main.labelselectchar=Виберіть вашого персонажа
main.experience=Досвід
main.charlevel=Рівень
main.gold=Золото
main.savingcharacter=Збереження...
main.backupcharacter=Створення резервної копії...
main.charclass=Клас
main.difficulty=Складність
main.attributesTab=Атрибути
main.skillsTab=Навички
main.gameNotDetected=Гру не виявлено
main.chooseGameDirectory=Будь ласка, виберіть теку, де встановлена гра
about.title=О {0}
about.version=Версія {0}
about.text=Права належать 2018 Емерсон Пінтер \ nЦе програмне забезпечення надається як є, немає ніяких гарантій на нього або на доступну або змінену інформацію. Завжди робіть резервну копію перед використанням. Це програмне забезпечення використовує бібліотеки з відкритим вихідним кодом, авторське право на додаткове програмне забезпечення належить відповідних власників.
about.newversion=Доступна нова версія, клікніть тут
alert.saveinprogress_header=Виконується файлова операція.
alert.saveinprogress_content=Виконується файлова операція, будь ласка зачекайте...
alert.savedbackedup_header=Збережено
alert.savedbackedup_content=Персонаж збережений і створена його резервна копія. Відкрийте теку \"{0}\" для перегляду резервної копії.
alert.errorloadingchar_header=Помилка завантаження персонажа
alert.errorloadingchar_content=Помилка читання атрибутів (health={0},energy={1},str={2},int={3},dex={4}). Це може бути через помилку, пошкодженого файлу персонажа або несумісності версії гри.
alert.errorbackup_header=Помилка створення резервної копії
alert.errorbackup_content=Сталася помилка під час створення резервної копії поточного персонажа в теку \"{0}\". Перевірте доступ до теки. Ваш персонаж не був змінений.
alert.errorsaving_header=Помилка збереження персонажа
alert.errorsaving_content=Сталася помилка під час збереження персонажа.
alert.errorcopying_header=Помилка копіювання персонажа
alert.errorcopying_content=Сталася помилка під час копіювання цього персонажа в теку \"{0}\". Перевірте доступ до теки. Ваш персонаж не був змінений.
difficulty.0=Норма
difficulty.1=Епос
difficulty.2=Легенда
skills.availableSkillPoints=Персонаж має {0} доступних очок навичок
skills.removeSkillsBefore=Вам необхідно відновити очки навичок перед маніпуляцією з майстерністю.
skills.reclaimSkillPoints=Прибрати навички
skills.reclaimMasteryPoints=Відновити {0} очок
skills.removeMastery=Прибрати
skills.masteryButton=Майстерність
alert.errorgamerunning_header=Закрийте гру для використання TQRespec
alert.errorgamerunning_content=Гра запущена, збереження і копіювання недоступно.
alert.changesinvaliddata=Некоректні дані \"{0}\"
```

**Comment by epinter** [comment]
Thanks !

**Comment by epinter** [comment]
Included in release v0.8.2.
